<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget_Test">
       <property name="gridVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_prevMonth">
         <property name="text">
          <string>Prev Month</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_today">
         <property name="text">
          <string>Today</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nextMonth">
         <property name="text">
          <string>Next Month</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_selectedDate">
         <property name="text">
          <string>Select Date Here</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_currentPage">
         <property name="text">
          <string>Current Page Here</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="check_AOD">
       <property name="text">
        <string>AOD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
